packages/initiatives/src/activate.ts

```diff
@@ -27,6 +27,7 @@ import { getProp } from "@esri/hub-common";
  * @param {any} groupIds hash of group props and ids
  * @param {IRequestOptions} requestOptions
  * @returns {Promise<IInitiativeModel>}
+ * @private
```

Comment:
What is the thinking behind this?

Reply:
At this point, Initiative activation is not used, and marking it private keeps it out of the docs. I've also gotten more draconian and the whole initiaives package is hidden from the docs... so... I guess this is not strictly needed